lib/pbench/cli/server/user_create.py

```diff
@@ -67,13 +70,20 @@ def execute(self):
     required=True,
     help="pbench server account last name (will prompt if unspecified)",
 )
+@click.option(
+    "--entitlement",
+    required=False,
+    default=None,
+    help="Optional entitlement of the user such as admin, if not specified defaults to None",
```

Comment:
But this still includes "defaults to None" in the help string, so what does it look like with the addition of `show_default`?

Also, I still would prefer this to be a keyword rather than just a string; e.g.,

```
   type=click.Choice(['ADMIN'], case_sensitive=False))
```

For bonus points, create the choice list directly from the ENUM rather than hardcoding it...

Reply:
"defaults to None" is still an awkward phrase, especially with "None" capitalized. If you don't specify a role, there are no roles enabled. "None" is not a role, nor are we presenting the possibility of specifying `--role=None`.

Further, your help already begins with "Optional"...

Oh, and it says "entitlement", which is obsolete. ;-)